### DISCUSSION TITLE
[Feature] Duplicate Detector: Link to Open Duplicate in Asset Viewer

### DISCUSSION THREAD

**Discussion by chk86** [immich-app/immich#10207] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

In the duplicate detector, it would be extremely useful when reviewing assets to be able to open any duplicate in a new asset viewer window/tab by clicking on each duplicate. 

Another option would be to click a duplicate and open a new window/tab to view the duplicate in the main timeline. 

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#10173
